src/common/variables.c

```diff
@@ -407,6 +410,14 @@ static char *_get_base_value(dt_variables_params_t *params, char **variable)
           || _has_prefix(variable, "EXIF_FOCAL_LENGTH"))
     result = g_strdup_printf("%.1f", params->data->exif_focal_length);
 
+  else if(_has_prefix(variable, "EXIF.FOCAL.LENGTH.EQUIV")
+          || _has_prefix(variable, "EXIF_FOCAL_LENGTH_EQUIV"))
```

Comment:
Not sure we want this, I had kept the all underscores values for upward compatibility but the favored syntax is with coma. So at this stage it is probably good to avoid introducing this.

Reply:
You're right, there is no need to ensure backward compatibility for new variables. From this point of view, alternative spellings with underlines are unnecessary.

My motives were as follows. For export, we have variable auto-completion, so there is no need to remember how to spell variable names. But there is no such possibility in other places, for example, in the watermark module. There we have to write the name of the variable exactly. Alternative spelling with underscores simply increases the likelihood that a user who does not remember very well how to spell variables, will write a name that we will accept.

Of course, the ideal solution would be to support autocompletion of variables in all places where they are supported.

Reply:
> Alternative spelling with underscores simply increases the likelihood that a user who does not remember very well how to spell variables, will write a name that we will accept.

Because of legacy naming... But we want to move away from this spelling. It is not even documented now. So better to avoid introducing more "error catching" code and have user learn about the proper spelling to me.

Reply:
OK. The code for the legacy spelling has been removed.